l10n_it_central_journal/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Author: Gianmarco Conte - Dinamiche Aziendali Srl
+# Copyright 2018
+# Dinamiche Aziendali Srl <www.dinamicheaziendali.it>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    'name': 'Italian Localization - Account central journal',
```

Comment:
Visto che non è traducibile, meglio mettere il nome in italiano
per es.
Account central journal → Libro giornale di/della contabilità

P.S.: c'è qualche riferimento riguardo la provenienza di "central journal" ? Non lo trovo fra i termini contabili in uso in lingua inglese.

Reply:
Anch'io direi "Libro giornale". Rif https://github.com/OCA/l10n-italy/pull/552#issuecomment-462856202

Reply:
Andata per Libro giornale, correggo